File: JavaScript/linearSearch.js
let linearSearch = (array, searchTerm) => {
    for(let i=0; i<array.length; i++){
        if(array[i] === searchTerm){
            return i;
        }
    }
    return "Not found."
};

console.log(linearSearch([9, 5, 14, 8, 42, 50], 14));
console.log(linearSearch([9, 5, 14, 8, 42, 50], 88));

File: Python/linearSearch.py
from operator import indexOf


def linearSearch(arr, searchTerm):
    for i in arr:
        if i == searchTerm:
            return arr.index(i)
    return -1

print(linearSearch([9, 5, 14, 8, 42, 50], 14))
print(linearSearch([9, 5, 14, 8, 42, 50], 88))